Doc/library/pdb.rst

```diff
@@ -123,6 +123,11 @@ The typical usage to inspect a crashed program is::
    0
    (Pdb)
 
+.. versionchanged:: 3.13
+   The implementation of :pep:`667` means that name assignments made via ``pdb``
+   will consistently and directly affect the active scope, even when running
```

Comment:
s/and directly//

Not sure that it is needed.

Reply:
Reading it again now, I think replacing the entire "consistently and directly" phrase with "immediately" will better convey the change that PEP 667 makes.